mapper.cpp

```diff
@@ -168,9 +168,9 @@ void parseDataFile(TileGenerator &generator, const string &input, string dataFil
 	// Check if input/../.. looks like a valid minetest directory
 	string minetestPath = input + PATH_SEPARATOR + ".." + PATH_SEPARATOR + "..";
 	string minetestConf = minetestPath + PATH_SEPARATOR + "minetest.conf";
-	int fd;
-	if (0 <= (fd = open(minetestConf.c_str(), O_RDONLY))) {
-		close(fd);
+	FILE * file;
+	if (0 <= (file = fopen(minetestConf.c_str(), O_RDONLY))) {
```

Comment:
``O_RDONLY`` -> ``"r"`` ...

As ``file`` is a pointer, please compare against ``NULL`` instead of ``0``.